Index: .gitignore
===================================================================
diff --git a/.gitignore b/.gitignore
deleted file mode 100644
--- a/.gitignore	(revision 50b4f9f591385a8e0bc8e3ea619a0a3461dc7b83)
+++ /dev/null	(revision 50b4f9f591385a8e0bc8e3ea619a0a3461dc7b83)
@@ -1,129 +0,0 @@
-# Byte-compiled / optimized / DLL files
-__pycache__/
-*.py[cod]
-*$py.class
-
-# C extensions
-*.so
-
-# Distribution / packaging
-.Python
-build/
-develop-eggs/
-dist/
-downloads/
-eggs/
-.eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-wheels/
-pip-wheel-metadata/
-share/python-wheels/
-*.egg-info/
-.installed.cfg
-*.egg
-MANIFEST
-
-# PyInstaller
-#  Usually these files are written by a python script from a template
-#  before PyInstaller builds the exe, so as to inject date/other infos into it.
-*.manifest
-*.spec
-
-# Installer logs
-pip-log.txt
-pip-delete-this-directory.txt
-
-# Unit test / coverage reports
-htmlcov/
-.tox/
-.nox/
-.coverage
-.coverage.*
-.cache
-nosetests.xml
-coverage.xml
-*.cover
-*.py,cover
-.hypothesis/
-.pytest_cache/
-
-# Translations
-*.mo
-*.pot
-
-# Django stuff:
-*.log
-local_settings.py
-db.sqlite3
-db.sqlite3-journal
-
-# Flask stuff:
-instance/
-.webassets-cache
-
-# Scrapy stuff:
-.scrapy
-
-# Sphinx documentation
-docs/_build/
-
-# PyBuilder
-target/
-
-# Jupyter Notebook
-.ipynb_checkpoints
-
-# IPython
-profile_default/
-ipython_config.py
-
-# pyenv
-.python-version
-
-# pipenv
-#   According to pypa/pipenv#598, it is recommended to include Pipfile.lock in version control.
-#   However, in case of collaboration, if having platform-specific dependencies or dependencies
-#   having no cross-platform support, pipenv may install dependencies that don't work, or not
-#   install all needed dependencies.
-#Pipfile.lock
-
-# PEP 582; used by e.g. github.com/David-OConnor/pyflow
-__pypackages__/
-
-# Celery stuff
-celerybeat-schedule
-celerybeat.pid
-
-# SageMath parsed files
-*.sage.py
-
-# Environments
-.env
-.venv
-env/
-venv/
-ENV/
-env.bak/
-venv.bak/
-
-# Spyder project settings
-.spyderproject
-.spyproject
-
-# Rope project settings
-.ropeproject
-
-# mkdocs documentation
-/site
-
-# mypy
-.mypy_cache/
-.dmypy.json
-dmypy.json
-
-# Pyre type checker
-.pyre/
Index: dummy-data-product/src/dependencies/utils/paths.yml
===================================================================
diff --git a/dummy-data-product/src/dependencies/utils/paths.yml b/dummy-data-product/src/dependencies/utils/paths.yml
deleted file mode 100644
--- a/dummy-data-product/src/dependencies/utils/paths.yml	(revision 50b4f9f591385a8e0bc8e3ea619a0a3461dc7b83)
+++ /dev/null	(revision 50b4f9f591385a8e0bc8e3ea619a0a3461dc7b83)
@@ -1,12 +0,0 @@
-metadata:
-    description: Contains relative paths to Google Cloud Storage Bucket for different sources to be harvested
-    bucket_name: psd_data
-
-paths:
-    <dummy_name>:
-        base_data_path: data/<dummy_name>/metadata/<dummy_name>_base_data.csv
-        master_data_path: data/<dummy_name>/raw/<dummy_name>_master_data.csv
-        cleaned_data_path: data/<dummy_name>/cleaned/<dummy_name>_cleaned_data.csv
-        geocoded_data_path: data/<dummy_name>/geocoded/<dummy_name>_geocoded_data.csv
-        standardized_data_path: data/<dummy_name>/standardized/<dummy_name>_standardized_data.csv
-        metadata_path: data/<dummy_name>/metadata/metadata.json
Index: dummy-data-product/src/client.py
===================================================================
diff --git a/dummy-data-product/src/client.py b/dummy-data-product/src/client.py
deleted file mode 100644
--- a/dummy-data-product/src/client.py	(revision 50b4f9f591385a8e0bc8e3ea619a0a3461dc7b83)
+++ /dev/null	(revision 50b4f9f591385a8e0bc8e3ea619a0a3461dc7b83)
@@ -1,54 +0,0 @@
-import dotenv
-import logging
-
-from datetime import datetime
-
-# Importing scraping and data processing modules
-# from dependencies.scraping.<file_name> import <class_name>
-# from dependencies.scraping.<file_name> import <class_name>
-# from dependencies.cleaning.<file_name> import <class_name>
-# from dependencies.geocoding.<file_name> import <class_name>
-# from dependencies.standardization.<file_name> import <class_name>
-
-dotenv.load_dotenv(".env")
-logging.basicConfig(level=logging.INFO)
-
-
-# In each step create an object of the class, initialize the class with 
-# required configuration and call the run method 
-def step_1():
-    logging.info("Scraped Metadata")
-
-
-def step_2():
-    logging.info("Scraped Main Data")
-
-
-def step_3():
-    logging.info("Cleaned Main Data")
-
-
-def step_4():
-    logging.info("Geocoded Cleaned Data")
-
-
-def step_5():
-    logging.info("Standardized Geocoded Data")
-
-
-if __name__ == "__main__":
-    import argparse
-
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--step", help="step to be choosen for execution")
-
-    args = parser.parse_args()
-
-    eval(f"step_{args.step}()")
-
-    logging.info(
-        {
-            "last_executed": str(datetime.now()),
-            "status": "Pipeline executed successfully",
-        }
-    )
Index: .vscode/settings.json
===================================================================
diff --git a/.vscode/settings.json b/.vscode/settings.json
deleted file mode 100644
--- a/.vscode/settings.json	(revision 50b4f9f591385a8e0bc8e3ea619a0a3461dc7b83)
+++ /dev/null	(revision 50b4f9f591385a8e0bc8e3ea619a0a3461dc7b83)
@@ -1,10 +0,0 @@
-{
-    "python.pythonPath": "ENV\\Scripts\\python.exe",
-    "python.formatting.provider": "black",
-    "python.linting.flake8Enabled": true,
-    "python.linting.enabled": true,
-    "python.linting.flake8Args": [
-        "--max-line-length=120",
-        "--ignore=E402,F841,F401,E302,E305,E266,E203,E713,W503,W605,E501,E741"
-    ],
-}
\ No newline at end of file
Index: data/sample.txt
===================================================================
diff --git a/data/sample.txt b/data/sample.txt
deleted file mode 100644
--- a/data/sample.txt	(revision 50b4f9f591385a8e0bc8e3ea619a0a3461dc7b83)
+++ /dev/null	(revision 50b4f9f591385a8e0bc8e3ea619a0a3461dc7b83)
@@ -1,3 +0,0 @@
-This a sample file placed in the *data* folder.
-
-All the outputs files from each step has to saved in this directory
\ No newline at end of file
diff --git a/dummy-data-product/src/dependencies/cleaning/cleaning.py b/dummy-data-product/src/dependencies/cleaning/cleaning.py
deleted file mode 100644
diff --git a/dummy-data-product/src/dependencies/standardization/__init__.py b/dummy-data-product/src/dependencies/standardization/__init__.py
deleted file mode 100644
diff --git a/dummy-data-product/src/dependencies/geocoding/geocoder.py b/dummy-data-product/src/dependencies/geocoding/geocoder.py
deleted file mode 100644
diff --git a/dummy-data-product/src/dependencies/geocoding/__init__.py b/dummy-data-product/src/dependencies/geocoding/__init__.py
deleted file mode 100644
diff --git a/dummy-data-product/src/dependencies/cleaning/__init__.py b/dummy-data-product/src/dependencies/cleaning/__init__.py
deleted file mode 100644
diff --git a/dummy-data-product/src/dependencies/standardization/standardizer.py b/dummy-data-product/src/dependencies/standardization/standardizer.py
deleted file mode 100644
diff --git a/dummy-data-product/src/requirements.txt b/dummy-data-product/src/requirements.txt
deleted file mode 100644
diff --git a/dummy-data-product/src/dependencies/scraping/__init__.py b/dummy-data-product/src/dependencies/scraping/__init__.py
deleted file mode 100644
diff --git a/dummy-data-product/src/dependencies/scraping/scraper.py b/dummy-data-product/src/dependencies/scraping/scraper.py
deleted file mode 100644
diff --git a/dummy-data-product/src/dependencies/utils/__init__.py b/dummy-data-product/src/dependencies/utils/__init__.py
deleted file mode 100644
